src/inverseRobot.js

```diff
@@ -5,9 +5,17 @@
  *
  * @return {object}
 */
-
 function inverseRobot(robot) {
-  // write code here
+  const newobj = {};
+
+  for (const prop in robot) {
+    if (newobj[robot[prop]] !== undefined) {
```

Comment:
would be better to use `hasOwnProperty` here
Also, would be better to move `robot[prop]` to the variable